common.blocks/i-bem-dom/i-bem-dom.js

```diff
@@ -312,6 +312,20 @@ function getJqueryCollection(html) {
 }
 
 /**
+ * Validates block to be class or specified description
+ * @param {*} Block Any argument passed to find*Block as Block
+ * @throws {Error} Will throw an error if the Block argument isn't correct
+ */
+function validateBlockParam(Block) {
+    if(
+        typeof Block === 'string' ||
+        typeof Block === 'object' && typeof Block.block === 'string'
+    ) {
+        throw new Error('Block must be a class or description (block, modName, modVal) of the block to find');
+    }
```

Comment:
- i still can pass primitives numbers, boolean, undefined 
 - any function

error message of this validation and validation description don't fit with code.